public/app/features/browse-dashboards/components/FolderActionsButton.tsx

```diff
@@ -86,11 +86,15 @@ export function FolderActionsButton({ folder }: Props) {
     );
   };
 
+  const managePermissions = t('browse-dashboards.folder-actions-button.manage-permissions', 'Manage permissions');
+  const move = t('browse-dashboards.folder-actions-button.move', 'Move');
+  const deleteFolderButton = t('browse-dashboards.folder-actions-button.delete', 'Delete');
```

Comment:
this is a **really** minor nit, but we should be consistent with our variable naming here. these are all the same thing, but one has `button` in and the other 2 do not. my suggestions:
- `managePermissionsLabel`
- `moveLabel`
- `deleteLabel`

Reply:
Thank you for suggestions, made the updates.

Reply:
i already approved, feel free to merge whenever 👍